### PR TITLE
Iterate dropdown

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**
Refactored dropdown component styles based on design input

### CODE REVIEW

**Comment by edgibbs** [comment]
Can one of the admins verify this patch?

**Comment by edgibbs** [comment]
retest this please

File: src/InputComponent.js [hunk 1]
```diff
@@ -14,11 +14,11 @@ const InputComponent = props => {
   return (
     <div className="form-group">
       <div
-        className={`${props.gridClassName}${props.serverError ||
-        props.validationError ||
-        props.required
-          ? 'has-error'
-          : ''}`}
+        className={`${props.gridClassName}${
```
Comment (edgibbs) [inline src/InputComponent.js]: Prettier doesn't like this formatting, shows an error locally when you run `yarn lint` .  Didn't show up in CI due to some weird config difference on Linux, which I'll check with Danny on.

File: stories/components/InputComponent.js [hunk 2]
```diff
@@ -40,15 +40,13 @@ const InputComponentStory = withInfo(
       labelClassName="Full Name"
```
Comment (edgibbs) [inline stories/components/InputComponent.js]: The example in storybook still doesn't hold onto the value when selected defaults back to 'Select ...'

File: src/InputComponent.js (hunk 1 above)
Comment (RamuKammagani-osi) [reply src/InputComponent.js]: It doesn't throw any error when i run lint and prettier in my local

Comment (edgibbs) [reply src/InputComponent.js]: It doesn't happen on the Linux image either, but when you run a straight prettier check it does fail: 
```yarn prettier:check```

Comment (edgibbs) [reply src/InputComponent.js]: So looks good on the prettier checks now.

**Comment by edgibbs** [comment]
Has some false positive warnings for CodeClimate, but we need to configure the duplication checks to avoid these.

**Pull request merged**